recipes/geofabrics/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "geofabrics" %}
+{% set version = "1.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geofabrics-{{ version }}.tar.gz
+  sha256: 6c14ece1404fadf0e1c5dca710e7d88a9454f0b43abd827cd15684e9bd48878b
+
+build:
+  noarch: python
+  entry_points:
+    - geofabrics_from_file = geofabrics.__main__:cli_run_from_file
+    - geofabrics_from_dict = geofabrics.__main__:cli_run_from_dict
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=64
+    - wheel
+    - pip
+  run:
+    - python >=3.9
+    - dask-core
+    - distributed
+    - gdal
+    - geopandas >=0.10
+    - python-pdal
+    - rioxarray
+    - geoapis
+    - netcdf4
+    - osmpythontools >=0.3.5
+    - tomli
+
+test:
+  imports:
+    - geofabrics
+  commands:
+    - pip check
```

Comment:
I'm not sure why pip check is failing with:

```
pip check
geofabrics 1.1.11 requires python-pdal, which is not installed.
```

We need to check why the package metadata is not finding it there. If you are sure this is working, it is ok to comment out pip check.

Reply:
Hmm, when it says "python-pdal, which is not installed." is this looking at what's on PyPI or whats specified in the package? 

You also say 'If I'm sure this is working?' What do you find is the best way to test locally? Sorry about all the questions?

Reply:
You can add a test case here or even run the full tests if they are shipped in the sdist. See the fiona feedstock for an example: https://github.com/conda-forge/fiona-feedstock/blob/main/recipe/meta.yaml#L69

Reply:
BTW, it is fine to drop pip-check in some cases and I believe this is one of them. If I understand it correctly there is no "python-gdal" namespace. So, as long as you have confidence in the other tests, it is fine to remove pip-check here so we can merge this.

Reply:
Thanks @ocefpaf and sorry about the months long silence from me. I was pulled onto some other tasks at work. I will be revisiting this now for the next few days and try get this wrapped up!

I will remove the pip-check and see what happens. I will also read up on running the tests (thanks for the fiona feedstock example) - it would be great to run the full test suite or at least one of the tests - but I'll just need to wrap my head around the sdist process to see if the test suit is shipped with it.

Reply:
@ocefpaf thanks for your patience with me on this case. I have removed the pip check. 

I currently do not include any tests with my dist as my test suite is quite large and I want to keep the sdist small for those installing. 

I was thinking I would let this first build become a feedstock and add a select test or two either to the sdist or to the feedstock. 

I'm also interested in your opinion on giving minimum limits to libraries in the toml. I am thinking of adding more as there are some old library releases I know won't work!

Reply:
thanks for merging!!